op-node/rollup/sequencing/sequencer.go

```diff
@@ -617,8 +617,6 @@ func (d *Sequencer) Init(ctx context.Context, active bool) error {
 	d.emitter.Emit(engine.ForkchoiceRequestEvent{})
 
 	if active {
-		// TODO(#11121): should the conductor be checked on startup?
```

Comment:
Answered in: https://github.com/ethereum-optimism/optimism/issues/11121#issuecomment-2366976557

Reply:
Just to copy things here:

This situation could happen, when starting sequencer, leadership transfer could happen and conductor could become a non-leader.

However, we don't need to do the check here because

there isn't really a way to guarantee that leadership does not transfer between the time we call startSequencer and it is actually started
and if the leadership transfer happened during that time, it is fine as sequencer won't be able to gossip its block out, and will shortly be turned off by a functioning conductor